docs/guides/nrfconnect_android_commissioning.md

```diff
@@ -0,0 +1,176 @@
+2. CHIPTool establishes a secure connection to the device over BLE and sends Thread credentials.
+3. The accessory device joins a Thread network.
+
+Since a typical smartphone doesn't have native support for Thread networking, extra effort is needed to prepare the test environment. The article describes how to turn a PC with spare Wi-Fi card and nRF 52840 DK into a Thread border router to forward Wi-Fi packets to a Thread network. The below diagram depicts connectivity between CHIPTool and Lock Example Application:
+
+```
+               +--------------------+
+               |    Smartphone      |
+     +---------|  Android CHIPTool  |---------+
+     |         +--------------------+         |
+     |                                        | Wi-Fi
+     |                                        |
+     |                                +---------------+  Ethernet  +----------+
+     |                                |      PC       |------------| Internet |
+     |                                +---------------+            +----------+
+     |                                        |
+ BLE |                                        | USB
+     |                                        |
+     |                                +---------------+
+     |                                | nRF 52840 DK  |
+     |                                | OpenThread RCP|
+     |                                +---------------+
+     |                                        |
+     |         +--------------------+         | Thread
+     |         |    nRF 52840 DK    |         |
+     +---------|  Lock Application  |---------+
+               +--------------------+
+```
+## Building and flashing OpenThread RCP firmware
+
+<details>
+<summary>Click here to expand</summary>
+<br>
+OpenThread RCP firmware is required to allow a PC to communicate with a Thread network. Complete the following steps to build and flash the RCP firmware onto a nRF 52840 DK:
+
+1. Download and install the
+[nRF Command Line Tools](https://www.nordicsemi.com/Software-and-Tools/Development-Tools/nRF-Command-Line-Tools).
+2. Clone OpenThread repository into the current directory:
+
+        $ git clone https://github.com/openthread/openthread.git
+        
+3. Install OpenThread dependencies:
+
+        $ cd openthread
+        $ ./script/bootstrap
+4. Set up build environment:
+
+        $ ./bootstrap
+5. Build OpenThread for the nRF 52840 DK:
+
+        $ make -f examples/Makefile-nrf52840
+
+6. Convert the resulting RCP image to HEX format:
+
+        $ arm-none-eabi-objcopy -O ihex output/nrf52840/bin/ot-rcp output/nrf52840/bin/ot-rcp.hex
+7. Flash the RCP firmware:
+
+        $ nrfjprog --chiperase --program output/nrf52840/bin/ot-rcp.hex --reset
+8. Disable Mass Storage feature on the device so that it doesn't interfere with core RCP functionalities:
+
+        $ JLinkExe
+        J-Link>MSDDisable
+        Probe configured successfully.
+        J-Link>exit
+</details>
+
```

Comment:
Add a note, that nRF52840 Dongle (PCA10059) might be used as well. To see how to build and flash the device please refer to https://github.com/openthread/openthread/blob/master/examples/platforms/nrf528xx/nrf52840/README.md#nrf52840-dongle-support-pca10059

Reply:
Added a note. Let me know if it's ok now.

Reply:
It's better, but now I realized that the two things are not clear:

- Full power cycle is required to have an effect.
- That this is one-time configuration (don't need to do that for every flashing).

Reply:
Followed your suggestions.